cypress/e2e/2-Locators-Actions/layout.cy.js

```diff
@@ -0,0 +1,19 @@
+describe("Verify stepper layout", () => {
+    it("Page /pages/layout/stepper", () => {
+      cy.visit("/pages/layout/stepper");
+  
+      // Функція для перевірки тексту на стеці та натискання кнопки "NEXT"
+    function checkStepText(stepText) {
+      cy.get(".step-content").should("contain", stepText);
+      cy.get("body > ngx-app > ngx-pages > ngx-one-column-layout > nb-layout > div > div > div > div > div > nb-layout-column > ngx-components > ngx-stepper > nb-card > nb-card-body > nb-stepper > div.step-content > button:nth-child(3)").click();
```

Comment:
Pls try this one 
cy.get('.appearance-filled:contains("next"):eq("0")').click()

 function checkStepText(stepText) {
        cy.get(".step-content").should("contain", stepText);
        stepText === "Step content #4"
        ?"Skip"
       : cy.get('.appearance-filled:contains("next"):eq("0")').click()
      }